src/internal/raw-interfaces.ts

```diff
@@ -296,3 +296,20 @@ export interface RawNftSale {
   bundleIndex: number;
   transactionHash: string;
 }
+
+export interface RawGetContractsForOwnerResponse {
+  contracts: RawContractForOwner[];
+  pageKey?: string;
+  totalCount: number;
+}
+
+export interface RawContractForOwner
+  extends Omit<RawNftContractMetadata, 'totalSupply' | 'openSea'> {
```

Comment:
good catch on this one too for the `opensea` spelling! Also note that `totalSupply` actually present in the response (example [link](https://eth-mainnet.g.alchemy.com/nft/v2/demo/getContractsForOwner?owner=vitalik.eth&pageSize=100))

Reply:
Ready!